plugins/modules/zfs_delegate_admin.py

```diff
@@ -80,20 +80,20 @@
 """
 
 EXAMPLES = r"""
-- name: Grant `zfs allow` and `unallow` permission to the `adm` user with the default local+descendents scope
+- name: Grant ZFS V(allow) and V(unallow) permission to the V(adm) user with the default local+descendents scope
```

Comment:
Ansible markup does not work in examples. Backticks or regular quotes are rendered as-is, that's why they are used. Whether backticks or regular single or double quotes are better is a matter of taste, I guess...